docs/_future/getting_started_future.md

```diff
@@ -56,8 +56,94 @@ In order to use the next-generation platform features, you'll need to accept a T
+2️⃣  Develop locally and see changes in real-time with: slack run
+
+3️⃣  When you're ready to deploy for production use: slack deploy
+
+🔔 If you leave the workspace, you won’t be able to manage any apps you’ve deployed to it. Apps you deploy will belong to the workspace even if you leave the workspace
+```
+
+Your app has now been created in the `my-app` directory! This application can be used as a template for you to build your own next-generation Bolt for Javascript app with. However, if you'd like to test the "Hello World" example included in your template, read on! 📖
+
+---
+### Set up your trigger {#setup-trigger}
+
+As mentioned, this app comes with pre-existing functionality - it uses Functions, Workflows and a Trigger set up to kick things off. Let's run a command to initialize that Trigger via the CLI.
+
+First, make sure you're in the project directory in your command line: `cd my-app`
+
+Then, run the following command to create a Trigger:
+```
+slack triggers create --trigger-def "triggers/hello-world-trigger.json"      
+```
+
+The above command will create a trigger for the selected workspace. Make sure to select the workspace you want and that it is appended by `(dev)`. This will create a dev instance of your app. Once the trigger is successfully created, you should see an output like this:
+
+```
+⚡ Trigger created
+   Trigger ID:   [ID]
+   Trigger Type: shortcut
+   Trigger Name: Sample Trigger
+   URL: https://slack.com/shortcuts/[ID]/[Some ID]
+```
+The provided URL will be what you use to run your Trigger. Copy this URL and save it somewhere; you'll need it for later.
+
+---
+### Run your app {#run-your-app}
+
+Now that your app and Trigger are successfully created, let's try running it!
+
+Run `slack run` to start up the app. Executing `slack run` starts a local development server, syncing changes to your workspace's development version of your app.
+
+You'll be prompted to select a workspace to install the app to&mdash;select the development instance of your workspace (you'll know it's the development version because the name has the string `(dev)` appended).
+
+> 💡 If you don't see the workspace you'd like to use in the list, you can `CTRL + C` out of the `slack run` command and run `slack auth login`. This will allow you to authenticate in your desired workspace to have it show up in the list for `slack run`.
+
+Once the app is successfully run, you'll see output in your Terminal to indicate it's running, similar to what you would see with any other Bolt for JavaScript app. You can search for the `⚡️ Bolt app is running! ⚡️` message to make sure that your app has successfully started up.
+
+### Trigger your app's workflow {#trigger-workflow}
+
+With your app running, access your workspace and paste the URL from the the Trigger you created in the previous step into a message in a public channel.
+
+> 💡 To make the trigger URL more widely accessible, we recommend saving the Trigger as a channel bookmark for easy access.
```

Comment:
So it gets auto saved in the `workflows` directory in the bookmarks bar now. We should update this message to reflect that

Reply:
Will do! I started the branch for additional updates, so added that in there!